support/ruby/collins-shell/Rakefile

```diff
@@ -62,3 +62,9 @@ YARD::Rake::YardocTask.new do |t|
   t.files = ['lib/**/*.rb']
   t.options = ['--markup', 'markdown']
 end
+
+begin
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
+rescue LoadError
```

Comment:
Why is rspec the only gem in this file guarded in such way? How is it possible to reach this step given the bundler error handling on top?

Reply:
I just copied this from the rspec documentation. I'll remove the guard.